External/smoke/CMakeLists.txt

```diff
@@ -0,0 +1,572 @@
+# Smoke
+# https://github.com/ROCm-Developer-Tools/aomp.git
+
+include(External)
+
+option(TEST_SUITE_SMOKE_FORCE_ALL "Execute all Smoke tests, even those known to be unsupported by Clang" OFF)
+
+set(TEST_SUITE_OFFLOADING_FLAGS --offload-arch=native CACHE STRING "Compiler arguments for OpenMP offloading")
+set(TEST_SUITE_OFFLOADING_LDFLAGS --offload-arch=native CACHE STRING "Linker arguments for OpenMP offloading")
+
+set(INTEL_TEST_SUITE_EXPECT_PASS)
+
+set(AMD_TEST_SUITE_EXPECT_PASS
+    alignedattribute
+    already_locked
+    ancestorC
+    aomp-issue376
+    assertok_error
+    assume_no_openmp
+    atomic
+    atomic_double
+    big-jump-loop-1
+    big-jump-loop-collapse
+    big-jump-loop-env-vars
+    big-jump-loop-matmul
+    big-jump-loop-split-2
+    big-jump-loop-split-3
+    big-jump-loop-split-4
+    bugz-50967
+    bugz-50967-c
+    bugz-50968
+    CanonNE
+    check_smoke.sh
```

Comment:
That is the driver used in our team to run the tests and check them for their results.

Reply:
Removed all the check scripts in commit #6c88027f